src/spacewar/ui/complex.cljc

```diff
@@ -109,7 +109,8 @@
     (q/text-font (:lcars (q/state :fonts)) 18)
     (let [{:keys [x y fps]} state
           fps (or fps 0)
-          fps (format "FPS: %6.2f" (float fps))]
+          fps #?(:clj (format "FPS: %6.2f" (float fps))
+                 :cljs (str "FPS: " (.toFixed fps 2)))]
```

Comment:
ClojureScript doesn't have a `format` function (which in Clojure delegates directly to the Java String format capability). So, instead we just use a little JavaScript interop to format the number with 2 significant figures after the decimal point.